applications/nsmrs/build/Dockerfile.nsmrs

```diff
@@ -0,0 +1,21 @@
+ARG VERSION=unspecified
+ARG VENDORING
+ARG GOPROXY
+ARG GO_VERSION
+
+FROM golang:${GO_VERSION}-alpine as build
+
+ENV GOPROXY=${GOPROXY}
+ENV PACKAGEPATH=github.com/networkservicemesh/networkservicemesh/applications/nsmrs
+ENV GO111MODULE=on
+
+RUN mkdir /root/networkservicemesh
+ADD [".","/root/networkservicemesh"]
+WORKDIR /root/networkservicemesh/applications/nsmrs
+RUN VENDORING=${VENDORING} ../../scripts/go-mod-download.sh
+
+RUN CGO_ENABLED=0 GOOS=linux go build ${VENDORING} -ldflags "-extldflags '-static' -X  main.version=${VERSION}" -o /go/bin/nsmrs ./cmd
```

Comment:
Looks like you already build application [here](https://github.com/networkservicemesh/networkservicemesh/pull/1672/files#diff-c31e810773a4c394261f8d76827aaca3R22) so why do we have to do it again in docker image?